app/services/google_search_service.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class GoogleSearchService
+  BASE_URL = 'https://www.google.com/search'
+
+  def initialize(keyword)
+    @url = "#{BASE_URL}?q=#{keyword}"
+    @user_agent = 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/605.1.15 (KHTML, like Gecko) '\
+                  'Version/11.1.2 Safari/605.1.15'
+  end
+
+  def call
+    HTTParty.get(@url, { headers: { 'User-Agent' => user_agent } })
```

Comment:
I think we can eliminate the `@` here. Just the same way we use the `user_agent`.

```suggestion
    HTTParty.get(url, { headers: { 'User-Agent' => user_agent } })
```

Reply:
Oh yes your right updated here https://github.com/aetheryna/google-scraper-ruby/pull/36/commits/5ccdd75db2c408cb66f542abf903918ab83b1750